IBF-Typhoon-model/post_test.py

```diff
@@ -0,0 +1,249 @@
+import os
```

Comment:
I think we should reuse the script in flood pipeline for get/post api 
https://github.com/rodekruis/IBF_FLOOD_PIPELINE/blob/master/pipeline/lib/flood_model/dynamicDataDb.py 
I will integrate this

Reply:
also realized there there are two versions for this script - i think we shouldn't push duplicate files to the repository

Reply:
sorry, please ignore this one. This should be removed. The correct function is in utility_fun